decidim-core/lib/decidim/abilities.rb

```diff
@@ -3,5 +3,6 @@
 module Decidim
   module Abilities
     autoload :ParticipatoryProcessRoleAbility, "decidim/abilities/participatory_process_role_ability"
+    autoload :UserManagerAbility, "decidim/abilities/user_manager_ability"
```

Comment:
It makes sense! I moved the new ability to the `app` folder so I can remove the `autoload` as well. Thanks 😄